triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -174,36 +167,37 @@ object Server {
       val jwt: Jwt = token._1
       val jwtPayload: JwtPayload = token._2
       val party: Party = Party(jwtPayload.party);
-      val uuid = UUID.randomUUID
-      val ident = uuid.toString
+      val triggerId = UUID.randomUUID
       val ref = ctx.spawn(
         TriggerRunner(
           new TriggerRunner.Config(
```

Comment:
There's quite a lot of options going in here now. There are roughly 3 or 4 parameter groups. It would be nice in the fullness of time to have a little more structure.